infra/base-images/base-libfuzzer/just_run

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Fuzzer runner.
+
+import configparser
+import os
+import subprocess
+import sys
+
+run_args = sys.argv[1:]
+
+# Use .options file if any.
+fuzzer_name = run_args[0]
+options_file = fuzzer_name + ".options"
```

Comment:
should this be checking for .options in /out, since the fuzzer binary is invoked without it's full path name?